go.mod

```diff
@@ -29,7 +29,7 @@ require (
 	github.com/google/uuid v1.1.1 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/influxdata/line-protocol v0.0.0-20180522152040-32c6aa80de5e
-	github.com/influxdata/pkg-config v0.2.5
+	github.com/influxdata/pkg-config v0.2.6
```

Comment:
Could you revert this change in your patch? It's probably one we should make, but it's unrelated to change needed here.

Reply:
If I remove this change, then the test suite fails to pass in CircleCI; or at least, it did last time. I can undo it and see if it still works.

Reply:
OK, no idea what was going on previously; tests passed this time with 0.2.5 no problem.